frontend/src/http/lists.js

```diff
@@ -86,4 +86,33 @@ async function updateListName(id, name) {
         return err.error;
     };
 }
-export { getLists, getList, addItem, addList, setCompletion, updateListName };
+
+// delete functions!!!
+async function deleteItem(listId, id) {
+    try {
+        const res = await fetch(`/api/todos/${listId}/${id}`,
+            {
+                method: 'DELETE'
+            })
+        return res;
+    } catch (err) {
+        console.log(err);
+        return err.error;
+    }
+}
+
+async function deleteList(listId) {
+    try {
+        const res = await fetch(`/api/todos/${listId}`,
+            {
+                method: 'DELETE'
+            })
+        if (res.status === 204) {
```

Comment:
If res is 204 then we return a string value but if it's not we return undefined? That doesn't seem correct?